sale_order_partner_invoice_period/models/res_partner.py

```diff
@@ -0,0 +1,17 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+from odoo import models, fields
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    period=fields.Selection(
```

Comment:
We should better define this field using a more specific name, like e.g.
```suggestion
    invoice_period = fields.Selection(
```